src/org/zaproxy/zap/extension/tokengen/resources/Messages.properties

```diff
@@ -46,6 +46,8 @@ tokengen.label.initialMessage   = <html>To generate and analyse tokens:\n\n<ul><
 tokengen.label.maxEntropy       = Maximum Entropy
 tokengen.panel.mnemonic			= t
 tokengen.panel.title            = Token Gen
+tokengen.optionspanel.name = Token Generator
+tokengen.optionspanel.option.threadsperscan = Number of threads:
 tokengen.results.table.header.timestamp.request = Req. Timestamp
```

Comment:
Could you make threads have a leading cap? `Threads`

Reply:
Oops shoulda been on line 50

Reply:
Changed.